src/commands/LoadController.php

```diff
@@ -185,31 +185,26 @@ protected function process($data)
         }
 
         $assignData = ArrayHelper::getValue($data, 'assign', []);
-        $findBy = ArrayHelper::remove($assignData, 'findBy');
+        $findBy = ArrayHelper::remove($assignData, 'findBy', 'findIdentity');
+
+        // Check function exist before execute
+        if(!method_exists(Yii::$app->user->identityClass, $findBy)) {
+            Yii::error(sprintf("Method %s doesn't exist", $findBy));
+            exit();
```

Comment:
We can use [`return ExitCode::UNSPECIFIED_ERROR;`](https://www.yiiframework.com/doc/api/2.0/yii-console-exitcode)

You can update the other exit to be return also.